(FM-8099) Update README to reflect status of types for XE devices (#330)
puppetlabs/cisco_ios: README.md

## Limitations

> We intend for support across the Catalyst family - both legacy IOS and IOS-XE. We have a finite set of physical platforms we perform continual testing against. In the event of errors or missing abstractions, users are requested to provide the error details (if present) via --debug --trace and a copy of the sanitized configuration (both Puppet and Cisco CLI).

Reviewer: ```suggestion
We intend for support across the Catalyst family — both legacy IOS and IOS-XE. We continually perform testing against a limited set of physical platforms. If you experience errors or missing abstractions, provide the error details (if present) via --debug --trace and a copy of the sanitized configuration (both Puppet and Cisco CLI).
```
I'm not sure what "We intend for support across the Catalyst family" means, do you support this now or is it something you plan to do in the future? If the latter then maybe don't say anything?

Author: Claire, this would be current functionality - IOS and IOS-XE are both catalyst.

Reviewer: Ah okay, maybe just "We support across the catalyst family - both legacy IOS and IOS-XE" then?  "Intend" makes it sound like you don't currently do this. Also lowercase for 'catalyst'.

Author: Changed to

'We support devices across the Catalyst family — both legacy IOS and IOS-XE.'

Author: On the bus but will change catalyst to lower case C

I think it was uppercase as it's a type of devices

https://en.m.wikipedia.org/wiki/Cisco_Catalyst

Reviewer: Ah okay, then upper case sounds good!
